cassandane/tiny-tests/JMAPContacts/contactgroup_set_cardgroup_get

```diff
@@ -0,0 +1,43 @@
+#!perl
+use Cassandane::Tiny;
+
+sub test_contactgroup_set_cardgroup_get
+{
+
+    my ($self) = @_;
+
+    my $jmap = $self->{jmap};
+
+    xlog $self, "create contacts";
+    my $res = $jmap->CallMethods([['Contact/set', {create => {
+                        "1" => { firstName => "foo", lastName => "last1" },
+                        "2" => { firstName => "bar", lastName => "last2" }
+                    }}, "R1"]]);
+    my $contact1 = $res->[0][1]{created}{"1"}{id};
+    my $contact2 = $res->[0][1]{created}{"2"}{id};
+
+    xlog $self, "create contact group";
+    $res = $jmap->CallMethods([['ContactGroup/set', {
+        create => {
+            "1" => {
+                name => "group1",
+                contactIds => [$contact1, $contact2]
+            }
+        }
+    }, "R2"]]);
+    $self->assert_not_null($res);
```

Comment:
nit: This line and the next 2 lines are unnecessary noise. We already have tests that assert that Cyrus adheres to the core spec of JMAP, there is no need to reassert this in every test.

Reply:
Right.  Fixed.